rtl/include/eclhelper.hpp

```diff
@@ -44,8 +44,8 @@ typedef unsigned short UChar;
 
 //Should be incremented whenever the virtuals in the context or a helper are changed, so
 //that a work unit can't be rerun.  Try as hard as possible to retain compatibility.
-#define ACTIVITY_INTERFACE_VERSION      699
-#define MIN_ACTIVITY_INTERFACE_VERSION  699             //minimum value that is compatible with current interface
+#define ACTIVITY_INTERFACE_VERSION      700
+#define MIN_ACTIVITY_INTERFACE_VERSION  700             //minimum value that is compatible with current interface
```

Comment:
Perhaps should leave some space for setting to 7.0.0 when 7.0 goes gold - maybe 650 would be a better number?

Reply:
Then we can include in the beta